src/app/shell/personal-cabinet/applications/applications.component.html

```diff
@@ -64,44 +64,48 @@
     <div class="tab-wrapper">
       <mat-tab-group mat-align-tabs="start" (selectedTabChange)="onTabChange($event)">
         <mat-tab label="Усі">
-          <ng-container *ngTemplateOutlet="ParentApplication; context:{$implicit: applications$ | async }">
+          <ng-container *ngTemplateOutlet="ParentApplication; context:{$implicit: applicationCard$ | async }">
           </ng-container>
         </mat-tab>
         <mat-tab label="{{applicationTitles[applicationStatus.Pending]}}">
-          <ng-container *ngTemplateOutlet="ParentApplication; context:{$implicit: applications$ | async }">
+          <ng-container *ngTemplateOutlet="ParentApplication; context:{$implicit: applicationCard$ | async }">
           </ng-container>
         </mat-tab>
         <!-- <mat-tab label="{{applicationTitles[applicationStatus.AcceptedForSelection]}}">
           <ng-container *ngTemplateOutlet="ProviderApplication; context:{$implicit: applications$| async }">
           </ng-container>
         </mat-tab> TODO: display when functionality is ready -->
         <mat-tab label="{{applicationTitles[applicationStatus.Approved]}}">
-          <ng-container *ngTemplateOutlet="ParentApplication; context:{$implicit: applications$ | async }">
+          <ng-container *ngTemplateOutlet="ParentApplication; context:{$implicit: applicationCard$ | async }">
           </ng-container>
         </mat-tab>
         <mat-tab label="{{applicationTitles[applicationStatus.Rejected]}}">
-          <ng-container *ngTemplateOutlet="ParentApplication; context:{$implicit: applications$ | async }">
+          <ng-container *ngTemplateOutlet="ParentApplication; context:{$implicit: applicationCard$ | async }">
           </ng-container>
         </mat-tab>
         <mat-tab label="{{applicationTitles[applicationStatus.Left]}}">
-          <ng-container *ngTemplateOutlet="ParentApplication; context:{$implicit: applications$ | async }">
+          <ng-container *ngTemplateOutlet="ParentApplication; context:{$implicit: applicationCard$ | async }">
           </ng-container>
         </mat-tab>
       </mat-tab-group>
     </div>
 
+  <app-paginator  [totalEntities]="(applicationsCard$ | async).totalAmount" [currentPage]="currentPage" [itemsPerPage]="applicationsPerPage$ | async"
+    (pageChange)="onPageChange($event)" (itemsPerPageChange)="onItemsPerPageChange($event)">
+  </app-paginator>
+
```

Comment:
I see you have added paginator only for parents, move it after the container at the very beginning for both roles